pkg/cmd/secret/delete/delete_test.go

```diff
@@ -144,6 +144,7 @@ func TestNewCmdDeleteBaseRepoFuncs(t *testing.T) {
 	tests := []struct {
 		name          string
 		args          string
+		env           map[string]string
```

Comment:
What's the reasoning for using a `map[string]string` here and not just explicitly setting the `GH_REPO` env var like below?

```suggestion
		ghRepoEnv           string
```

Reply:
Truthfully, I was borrowing from similar patterns in other tests as well as the idea of putting the context in the testing table rather than the code.

Additionally, I think I was influenced by https://github.com/cli/cli/pull/10535 where the draft checked for the existing of `GH_REPO` environment variable rather than the value of it.  Thus this test is only setting environment variables and values intentionally as there is a likelihood it will be important.

Reply:
https://github.com/cli/cli/blob/b8cf16dc2d4d8040470beba883847d002119925a/pkg/cmdutil/auth_check_test.go#L12-L57

Reply:
Yeah, I can see value in both approaches. The only issue is the generic `map[string]string` masks what env vars we'll actually be testing. This gets disambiguated for any test that requires env vars in the test definition, but not elsewhere.

Again, this is a tiny optimization, so I'll leave the decision to you 🙂